src/renderEmail.js

```diff
@@ -1,6 +1,12 @@
 import ReactDOMServer from 'react-dom/server'
 
-export default function renderEmail(emailComponent) {
+export default function renderEmail(emailComponent, { renderMailchimpAttr } = {}) {
   const doctype = '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">'
-  return doctype + ReactDOMServer.renderToStaticMarkup(emailComponent)
+  let markup = doctype + ReactDOMServer.renderToStaticMarkup(emailComponent)
+
+  if (renderMailchimpAttr) {
```

Comment:
Understood and that makes sense. How do you feel about utils we can wrap around our markup string?

ie: renderMailChimpAttr( renderEmail(<Component />) )